server/src/test/java/io/crate/execution/dml/IndexerTest.java

```diff
@@ -1200,6 +1199,7 @@ public void test_indexing_boolean_results_in_same_fields_as_document_mapper_if_n
     }
 
     @Test
+    @Ignore(value = "We don't support dynamic creation of nested arrays due to translog restrictions")
```

Comment:
The scenario this test was checking did not in fact work previously, as the dynamically added nested array column would cause an error to be thrown when the mapping updates were sent to the master cluster update thread.  With this PR the error is thrown earlier, during the `collectSchemaUpdates()` method.